cf-agent/verify_files_utils.c

```diff
@@ -3649,27 +3854,17 @@ static int VerifyFinderType(EvalContext *ctx, const char *file, const Attributes
 
     memset(&fndrInfo, 0, sizeof(fndrInfo));
 
-    getattrlist(file, &attrs, &fndrInfo, sizeof(fndrInfo), 0);
+    getattrlist(changes_file, &attrs, &fndrInfo, sizeof(fndrInfo), 0);
 
     if (fndrInfo.fi.fdType != *(long *) a->perms.findertype)
     {
         fndrInfo.fi.fdType = *(long *) a->perms.findertype;
 
-        switch (a->transaction.action)
+        if (MakingChanges(ctx, pp, a, result,
+                          "set Finder Type code of '%s' to '%s'", file, a->perms.findertype))
```

Comment:
Previously all branches of the switch (including default) returned something. But now, it only returns inside the if, no else.